db/re/item_combo_db.txt

```diff
@@ -2526,3 +2526,46 @@
 // 400151:560017,{ bonus bMaxHPrate,15; bonus2 bSkillAtk,"SR_TIGERCANNON",12*(getequiprefinerycnt(EQI_HAND_R)/2); }
 // 400151:590020,{ bonus2 bAddSize,Size_All,15; bonus2 bMagicAddSize,Size_All,15; bonus2 bSkillAtk,"AB_DUPLELIGHT",12*(getequiprefinerycnt(EQI_HAND_R)/2); }
 // 400151:610019,{ bonus bDelayrate,-15; bonus2 bSkillAtk,"GC_ROLLINGCUTTER",12*(getequiprefinerycnt(EQI_HAND_R)/2); }
+490122:490123,{ bonus bFixedCast,-200; if (BaseLevel >= 105) { bonus2 bAddClass,Class_All,1; bonus bAspdRate,2; } if (BaseLevel >= 110) { bonus2 bAddClass,Class_All,1; bonus bAspdRate,3; } if (BaseLevel >= 120) { bonus2 bAddClass,Class_All,1; bonus bAspdRate,5; } }
+490124:490125,{ bonus bFixedCast,-200; if (BaseLevel >= 105) { bonus bMatkRate,1; bonus bVariableCastrate,-2; } if (BaseLevel >= 110) { bonus bMatkRate,1; bonus bVariableCastrate,-3; } if (BaseLevel >= 120) { bonus bMatkRate,1; bonus bVariableCastrate,-5; } }
+490126:490127,{ bonus bFixedCast,-200; bonus bAspdRate,10; if (BaseLevel >= 105) { bonus2 bAddClass,Class_All,2; } if (BaseLevel >= 110) { bonus bShortAtkRate,5; bonus bLongAtkRate,5; } if (BaseLevel >= 120) { bonus bCritAtkRate,5; } }
+490128:490129,{ bonus bFixedCast,-200; bonus bVariableCastrate,-10; if (BaseLevel >= 105) { bonus bMatkRate,2; } if (BaseLevel >= 110) { bonus2 bMagicAtkEle,Ele_All,5; } if (BaseLevel >= 120) { bonus2 bMagicAddEle,Ele_All,5; } }
+15362:4123,{ bonus3 bAddEffWhenHit,Eff_Stun,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Freeze,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Stone,5000,ATF_MAGIC; }
+15362:4381,{ bonus bDelayrate,-5; bonus bAspd,1; }
+15367:4141,{ .@r = min(14,getequiprefinerycnt(EQI_ARMOR)); bonus bMaxHPrate,3*(.@r/2); }
+15367:4363,{ .@r = getrefine(); if (.@r>=10) { bonus bMaxHPrate,min(14,.@r)-5; } }
```

Comment:
For me the hp max increases from (refine-5)*(refine-5) with refine up to 14
```suggestion
15367:4363,{ .@r = getrefine(); if (.@r>=10) { .@r = min(14,.@r)-5; bonus bMaxHPrate,pow(.@r,2); } }
```

Reply:
Thanks